en/1.7.0/guide/dashboard/index.md

```diff
@@ -6,148 +6,49 @@ title: Dashboard
-          "timestamp": "2015-10-06T16:00:43+02:00",
-          "device": "Android"
-        },
-        "header": {
-          "environmentalInfo": "from sync(...) method, 1 per upload"
-        },
-        "body": [
-          { "yourTrace": "from save(...) method, 1 per trace" },
-          { "yourTrace": "another saved trace" },
-        ]
-      }, {
-        ...
-      }
-    ]
-
-### Filter uploaded data
-
-If you want to insert some specifics, calculated values,
-you can add a pre-upload filter, applied on each uploaded data (see the section above for the syntax).
-
-    // This filter will process every uploaded data
-    rest.setPreUploadTreatment(function(data) { // data will be the uploaded JSON
-       var myUploadedDataArray = data.body;
-       var myMetadata = data.metadata;
-       myMetadata.count = myUploadedDataArray.length;
-       return data; // It is recommended to send back the same json structure as the input.
-    });
-
-### Filter output
-
-If you don't want to download every collected traces from the server, you can define custom routes to filter the data beforehand.
-
-The creation of these routes can be done in the `Filters` menu, from the collect dashboard.
-In this menu, you will find a script editor, in which you can add routes with the following (Javascript) syntax:
-
-    // This filter will only return the metadata of each collected trace.
-    // It can be accessed from the route /api/v1/$collectIdentifier/data/meta
-    rest.prepareFilter("meta", function(data){ // data will be your raw data json array
-      var result = [];
-      for each (var ele in data) {
-        result.push(ele.metadata);
-      }
-      return result; // This will be parsed as a Json.
-    });
-
-
-After saving your script, you can access the result from the route `/api/v1/$cropIdentifier/data/meta`.
+### Code it
 
-## Retrieve your media
+You can then use the editor to write your crop's script using every [Sting](../../stings) you need.
+It should be quite easy and user friendly but if you're in trouble, you can contact us at [contact@apisense.com](contact@apisense.com)
 
-Some stings will record media files (like pictures, sound, or video).
-For obvious reasons, those media will not directly be saved in the data JSON,
-but an identifier will be set instead.
+### Publish it
 
+When ready to start collecting data, press the _deploy_ button on the editor.
+Your crop will be activated and published on the store, waiting for users to subscribe to it.
 
-### Metadata format:
+## Manage own applications
 
-You can retrieve medatada for every media uploaded on a crop from the route `/api/v1/$cropIdentifier/media`.
-Every metadata object will contain the following elements:
+When creating a custom application using APISENSE, you will need to [provide an access key to your SDK](../sdk#authorize-your-sdk).
+This key will be generated on the add of the application on the [dashboard interface](https://apisense.io/settings/apps).
```

Comment:
This key will be generated within / at the same time of your application creation on the..